server/routes/findInterventions/searchResultsPresenter.ts

```diff
@@ -0,0 +1,80 @@
+import { Eligibility, Intervention } from '../../services/interventionsService'
+import { SummaryListItem } from '../../utils/summaryList'
+import utils from '../../utils/utils'
+
+export interface SearchResultPresenter {
+  title: string
+  href: string
+  body: string
+  summary: SummaryListItem[]
+}
+
+export default class SearchResultsPresenter {
+  constructor(private readonly interventions: Intervention[]) {}
+
+  readonly results: SearchResultPresenter[] = this.interventions.map(intervention => ({
+    title: intervention.title,
+    href: '#',
+    body: intervention.description,
```

Comment:
This was hardcoded to a string description before (I force pushed before leaving a comment, but have updated it, and the mocks).